client/widgets/Buttons.cpp

```diff
@@ -425,6 +437,17 @@ void CToggleGroup::setSelected(int id)
 	selectionChanged(id);
 }
 
+void CToggleGroup::setSelectedOnly(int id)
+{
+	for (map<int, shared_ptr<CToggleBase> >::iterator it = buttons.begin(); it != buttons.end(); it++)
```

Comment:
No space after `for` please and use `auto` in iterator declaration.

Reply:
Make sense.